tests/v2/validation/clustertemplates/provisioning/README.md

```diff
@@ -0,0 +1,72 @@
+## Cluster template Provisioning Config
+
+For your config, you will need everything in the [Prerequisites](../README.md) section on the previous readme along with and at least one [Cloud Credential](#cloud-credentials). 
+
+Your GO test_package should be set to `provisioning/clustertemplates`.
+Your GO suite should be set to `-run ^TestClusterTemplateRKE1ProvisioningTestSuite$`.
+Please see below for more details for your config. Please see below for more details for your config. Please note that the config can be in either JSON or YAML (all examples are illustrated in YAML).
+
+## Table of Contents
+1. [Getting Started](#Getting-Started)
+
+## Getting Started
+See below a sample config file to run this test:
+```yaml
+rancher:                                   
+  host: ""                                  #required
+  adminToken: ""                            #required
+  clusterName: ""                           
+
+provisioningInput:
+  etcdRKE1:
+    backupConfig:
+      s3BackupConfig:
+        accessKey: <AWS Access Key>
+        secretKey: <AWS Secret Key>
+        endpoint: <S3 endpoint>
+        region: <S3 bucket region>
+        bucketName: <S3 bucket name>
+  RKE1KubernetesVersions: ["<Your preferred Version>"]    #optional. if empty, latest version is considered.
+  cni: ["calico"]                                  #optional, if empty, calico is considered.
+  nodeProviders: ["aws"]                           #required (at least 1)                    
```

Comment:
note - nodeProviders is for custom clusters, providers should be for provisioned clusters.